theme.css

```diff
@@ -0,0 +1,3 @@
+body{
```

Comment:
This is fine, but curious if the task was just to add a background to this page?